sparrow-py/pysrc/sparrow_py/expr.py

```diff
@@ -285,3 +299,12 @@ def run_to_csv_string(self) -> str:
         """Execute the expression and return the CSV as a string."""
         # Convert PyArrow -> Pandas -> CSV, omitting the index (row).
         return self.run().to_csv(index=False)
+
+def _aggregation(op: str, input: Expr, window: Optional[Window]) -> Expr:
+    """Helper for creating an aggregation."""
+    if window is None:
+       return Expr.call(op, input, None, None)
+    elif isinstance(window, SinceWindow):
+       return Expr.call(op, input, window._predicate, 1)
```

Comment:
Can you change the `1` to `None`? This will ensure we actually use the `since` window implementations, which should be more performant than the TwoStacks. (Also, collect only supports `since`)

Ideally we pass the type of the window rather than relying on "number of non-null args" to determine window type..but, future change.